src/index.jsx

```diff
@@ -44,20 +115,31 @@ class Container extends Component {
           zoomMin={scale.zoomMin}
           zoomMax={scale.zoomMax}
         />
-        <div className={`layout ${isOpen ? 'is-open' : ''}`}>
-          <div className="layout__side">
+        <div className={`layout ${isOpen ? 'is-open' : ''}`} ref={(layout) => { this.layout = layout }}>
```

Comment:
i'm Not sure what you mean? as I am referencing that node in the component.  How would a prop enable me to reference it?